### ISSUE TITLE
Storage root must match that calculated

### ISSUE THREAD

**Issue by soc1c** [paritytech/polkadot#420]
Restart after #419 causes

```
Hash not equal
c788724dafa5ada309c55f6235413fcea8825bec3259b3ce69e3ba1fc58f9377
2849d4064e84f951cb7686cdb9a45726a8bebfb357b264edc7f50b29c0a3707f
panicked at 'Storage root must match that calculated.', /Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3
2019-09-06 18:42:55 Block import error:
Execution(Wasmi(Trap(Trap { kind: Unreachable })))
Hash not equal
c788724dafa5ada309c55f6235413fcea8825bec3259b3ce69e3ba1fc58f9377
2849d4064e84f951cb7686cdb9a45726a8bebfb357b264edc7f50b29c0a3707f
panicked at 'Storage root must match that calculated.', /Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3
2019-09-06 18:42:56 Block import error:
Execution(Wasmi(Trap(Trap { kind: Unreachable })))
Hash not equal
c788724dafa5ada309c55f6235413fcea8825bec3259b3ce69e3ba1fc58f9377
2849d4064e84f951cb7686cdb9a45726a8bebfb357b264edc7f50b29c0a3707f
panicked at 'Storage root must match that calculated.', /Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3
2019-09-06 18:42:56 Block import error:
Execution(Wasmi(Trap(Trap { kind: Unreachable })))
Hash not equal
c788724dafa5ada309c55f6235413fcea8825bec3259b3ce69e3ba1fc58f9377
2849d4064e84f951cb7686cdb9a45726a8bebfb357b264edc7f50b29c0a3707f
panicked at 'Storage root must match that calculated.', /Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3
2019-09-06 18:42:57 Block import error:
Execution(Wasmi(Trap(Trap { kind: Unreachable })))
2019-09-06 18:42:57 Discovered new external address for our node: /ip4/10.0.1.77/tcp/34567/p2p/QmU4ahXCJBvZ8adrFGdEES2iyfQkjVwUEh5dWNrvaCt48H
Hash not equal
c788724dafa5ada309c55f6235413fcea8825bec3259b3ce69e3ba1fc58f9377
2849d4064e84f951cb7686cdb9a45726a8bebfb357b264edc7f50b29c0a3707f
panicked at 'Storage root must match that calculated.', /Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3
2019-09-06 18:42:57 Block import error:
Execution(Wasmi(Trap(Trap { kind: Unreachable })))
2019-09-06 18:42:57 Idle (11 peers), best: #203647 (0x50eb…7994), finalized #203646 (0x1b2c…3d27), ⬇ 46.5kiB/s ⬆ 13.3kiB/s
Hash not equal
c788724dafa5ada309c55f6235413fcea8825bec3259b3ce69e3ba1fc58f9377
2849d4064e84f951cb7686cdb9a45726a8bebfb357b264edc7f50b29c0a3707f
panicked at 'Storage root must match that calculated.', /Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3
2019-09-06 18:42:58 Block import error:
Execution(Wasmi(Trap(Trap { kind: Unreachable })))
Hash not equal
c788724dafa5ada309c55f6235413fcea8825bec3259b3ce69e3ba1fc58f9377
2849d4064e84f951cb7686cdb9a45726a8bebfb357b264edc7f50b29c0a3707f
panicked at 'Storage root must match that calculated.', /Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3
2019-09-06 18:42:58 Block import error:
Execution(Wasmi(Trap(Trap { kind: Unreachable })))
Hash not equal
c788724dafa5ada309c55f6235413fcea8825bec3259b3ce69e3ba1fc58f9377
2849d4064e84f951cb7686cdb9a45726a8bebfb357b264edc7f50b29c0a3707f
panicked at 'Storage root must match that calculated.', /Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3
2019-09-06 18:42:59 Block import error:
Execution(Wasmi(Trap(Trap { kind: Unreachable })))
```

Running latest 0.5.1: 60155e89

```
[user@host ~]$ polkadot --version
polkadot 0.5.1-60155e89-x86_64-linux-gnu

[user@host ~]$ uname -a
Linux 5.2.11-arch1-1-ARCH #1 SMP PREEMPT Thu Aug 29 08:09:36 UTC 2019 x86_64 GNU/Linux
```

**Comment by soc1c** [comment]
Where does `/Users/gav/.cargo/git/checkouts/substrate-7e08433d4c370a21/3e975b3/srml/executive/src/lib.rs:319:3` come from?

**Comment by bkchr** [comment]
Duplicate of: https://github.com/paritytech/polkadot/issues/413
